### ISSUE TITLE
Keyboard shortcuts don't work in macOS - even if Plover is in front

### ISSUE THREAD

**Issue by Achim63** [openstenoproject/plover#787]
Classification: UI/Usability

Reproducibility: Always

## Summary

The keyboard shortcuts in the "tray" menu are not usable on macOS.

## Steps to Reproduce

Try to use e.g. Cmd+K while Plover is the active application.

## Expected Results

Shortcuts work as expected.
Plover should have a regular menu that makes use of those shortcuts. The shortcuts in the "tray" menu should be deleted as they will never work.

## Actual Results

nothing happens ...

## Version

Plover version 4.0.0.dev1+20.g60a373f.

Installed via: dmg

## Configuration

OS:
macOS 10.12.5

**Comment by morinted** [comment]
Works for me. Not sure what command K is supposed to do, but ⌘O prompts to add a dictionary and ⌘T opens the Paper Tape. I tried this with Plover as the focused application and while having the tray open.

**Comment by Achim63** [comment]
For the 4.0 build on Sierra? None of the shortcuts work for me, I have to select them using a mouse.

**Comment by morinted** [comment]
Have you changed your modifier keys perchance? It could affect the Plover shortcuts.

<img width="780" alt="screen shot 2017-07-10 at 11 15 12 am" src="https://user-images.githubusercontent.com/5840970/28025346-281259e4-6561-11e7-8533-ed6b35ecf8c3.png">

Yes, I'm on Sierra with Plover 4.x

**Comment by Achim63** [comment]
Yes, I have, as I'm using a PC keyboard at work I swapped Option and Command. But resetting it to the defaults doesn't help. I also tried quitting Keyboard Maestro, which might interfere as well. All to no avail. I'll try again at home with a standard Mac keyboard (but last time I tried the shortcuts didn't work there either).

**Comment by Achim63** [comment]
So comparing with my iMac at home, things get even stranger. Here I see the File and Edit menus of Plover, that weren't even there on the older iMac at work. Cmd+O does work. The shortcuts on the right hand menu (the one with the Plover icon, no idea what those are called ...) don't work at all (Cmd+N/L/T/J), which I would expect since it is global and would overwrite any app's shortcuts - that's why I proposed to move/copy those entries to the main menu on the left.

**Comment by Achim63** [comment]
I examined this issue a bit closer now - the standard menus with shortcuts only appear if Plover's main window is opened (I have it set to "start minimized"). Including the Tools menu with (working) shortcuts. When I close the main window, sometimes the Tools menu disappears, sometimes it stays but all menu items are greyed out (and once that happens, I have to restart Plover to make it work again).
I think the complete menu should be available at all times, no matter if the main window is open.

**Comment by mkrnr** [comment]
Closing this ticket due to inactivity. Please comment if this is still an issue.
